langchain/chains/sql_database/base.py

```diff
@@ -91,7 +91,7 @@ def _call(self, inputs: Dict[str, Any]) -> Dict[str, Any]:
         intermediate_steps: List = []
         try:
             intermediate_steps.append(llm_inputs)  # input: sql generation
-            sql_cmd = llm_chain.predict(**llm_inputs).strip(" '\"")
+            sql_cmd = llm_chain.predict(**llm_inputs).strip()
```

Comment:
I found strip didn't work for me, it removed only one, that's why I used replace (thinking that it'll be a single quote within)

imho, your point about maybe being used is valid and would be better to have strip.

these are some options (found here: https://stackoverflow.com/questions/40950791/remove-quotes-from-string-in-python)

```
# or, if they only occur at start and end...
a = a.strip('"')
'sajdkasjdsak" "asdasdasds'

# or, if they only occur at start...
a = a.lstrip('"')

# or, if they only occur at end...
a = a.rstrip('"')
```

# or, if they only occur at start and end...
a = a.strip('\"')
'sajdkasjdsak" "asdasdasds'

# or, if they only occur at start...
a = a.lstrip('\"')

# or, if they only occur at end...
a = a.rstrip('\"')